WordPress/Classes/ViewRelated/Domains/Domain registration/DomainPurchasingWebFlowController.swift

```diff
@@ -0,0 +1,338 @@
+import UIKit
+import AutomatticTracks
+import Sentry
+
+final class DomainPurchasingWebFlowController {
```

Comment:
This logic was mostly extracted from `RegisterDomainSuggestionsViewController` with few modifications, but it would be great to review this logic again.